paf-mvp-core-js/src/user-agent.ts

```diff
@@ -1,6 +1,6 @@
 // FIXME Should be more elaborate. For the moment just consider Safari doesn't support 3PC
-import { Browser } from 'detect-browser';
+import { IBrowser } from 'ua-parser-js';
```

Comment:
Why roll back? Is it a typo in a merge?

Reply:
This has come from the main branch where UA-parser-js was replaced with detect-browser. Not related to this PR.